src/bin/cobalt/serve.rs

```diff
@@ -39,7 +46,8 @@ pub fn serve_command_args() -> clap::App<'static, 'static> {
 
 pub fn serve_command(matches: &clap::ArgMatches) -> Result<()> {
     let port = matches.value_of("port").unwrap().to_string();
-    let ip = format!("127.0.0.1:{}", port);
+    let host = matches.value_of("host").unwrap().to_string();
+    let ip = format!("{}:{}", host, port);
```

Comment:
Maybe its just me but it reads better here and above to have `host` before `port`.  Mind swapping the order?